components/ws-manager-bridge/src/bridge.ts

```diff
@@ -299,6 +311,7 @@ export class WorkspaceManagerBridge implements Disposable {
 
             log.info({instanceId, workspaceId: instance.workspaceId}, "Database says the instance is starting for too long or running, but wsman does not know about it. Marking as stopped in database.", {installation});
             instance.status.phase = "stopped";
+            instance.stoppingTime = new Date().toISOString();
```

Comment:
Maybe a nit, but: guaranteed `stoppingTime == stoppedTime` is a nice thing to have for downstream usage: analytics, bug hunting, ad-hoc DB fixes, etc.